metron-deployment/roles/metron_streaming/tasks/main.yml

```diff
@@ -15,25 +15,18 @@
 #  limitations under the License.
 #
 ---
-- name: Create Metron streaming directories
-  file: path="{{ metron_directory }}/{{ item.name }}"  state=directory mode=0755
+- name: Create Metron directories
+  file: path={{ metron_directory }}/{{ item }} state=directory mode=0755
   with_items:
-      - { name: 'lib'}
-      - { name: 'bin'}
-      - { name: 'config'}
+      - lib
+      - bin
+      - config
 
 - name: Create Source Config Directory
-  file:
-    path: "{{ zookeeper_config_path }}"
-    state: directory
-    mode: 0755
+  file: path={{ zookeeper_config_path }} state=directory mode=0755
```

Comment:
Not a strong objection, but I do find the vertical presentation easier to read. I think it's because the yaml file decorator does a better job calling out the parameters in the vertical orientation.